src/CognitiveServices/CognitiveServices/help/Az.CognitiveServices.md

```diff
@@ -26,9 +26,6 @@ Get Deployment of a Cognitive Services account
 ### [Get-AzCognitiveServicesAccountKey](Get-AzCognitiveServicesAccountKey.md)
 Gets the API keys for an account.
 
-### [Get-AzCognitiveServicesAccountModel](Get-AzCognitiveServicesAccountModel.md)
```

Comment:
Please also re-generated this file as I did not see the new cmdlets and not sure if these cmdlets need to be removed.